runner/hash.go

```diff
@@ -30,12 +30,12 @@ func GetHash(path string) string {
 		if err != nil {
 			panic(err)
 		}
-		defer f.Close()
 
 		if _, err := io.Copy(h, f); err != nil {
 			log.Fatal(err)
 		}
 
+		f.Close()
```

Comment:
Don't we want to use defer here? If the io.Copy() fails we don't close the input file now

Reply:
Fixed by explicitly closing there too. Not great but it closes the technically incorrect defer